CHANGELOG.md

```diff
@@ -6,3 +6,7 @@ This file is used to list changes made in each version of the rvm_fw cookbook.
 0.1.0
 -----
 - [Steven Haddox] - Initial release of rvm_fw
+
+0.2.0
```

Comment:
This was correct at `0.5.0`. Thought I deleted the comment where I put the wrong version, sorry :(